common/djangoapps/status/tests.py

```diff
@@ -72,18 +82,16 @@ def remove_status_file(self):
     def tearDown(self):
         self.remove_status_file()
 
-    def test_get_site_status_msg(self):
+    @ddt.data(*checks)
+    @ddt.unpack
+    def test_get_site_status_msg(self, json_str, exp_none, exp_toy, exp_full):
         """run the tests"""
-        for (json_str, exp_none, exp_toy, exp_full) in self.checks:
-
-            self.remove_status_file()
-            if json_str:
-                self.create_status_file(json_str)
-
-            print "checking results for {0}".format(json_str)
-            print "course=None:"
-            self.assertEqual(get_site_status_msg(None), exp_none)
-            print "course=toy:"
-            self.assertEqual(get_site_status_msg(self.toy_id), exp_toy)
-            print "course=full:"
-            self.assertEqual(get_site_status_msg(self.full_id), exp_full)
+
+        self.remove_status_file()
+        if json_str:
+            self.create_status_file(json_str)
+
+        for course_id, expected_msg in [(None, exp_none), (self.toy_id, exp_toy), (self.full_id, exp_full)]:
+            self.assertEqual(get_site_status_msg(course_id), expected_msg)
+            self.assertEqual(cache.get('site_status_msg'), expected_msg)
```

Comment:
@sarina , if you're on devstack, you can add a `status_message.json` file to the same directory as `lms.env.json` with contents like:
```json
{
  "global" : "Hello, Globe"
}
```
Then, when you view courseware in LMS, you'll see the status message. I'm not sure how to manually check that the cache is working, other than changing that file and waiting 5 minutes :)